lighthouse-core/audits/dobetterweb/uses-http2.js

```diff
@@ -47,11 +48,14 @@ class UsesHTTP2Audit extends Audit {
         const requestHost = new URL(record._url).host;
         return requestHost === finalHost;
       }).map(record => {
+        if (urls.has(record._url)) return;
+        urls.add(record._url);
+
         return {
           protocol: record.protocol,
-          url: record.url, // .url is a getter and not copied over for the assign.
+          url: record._url,
         };
-      });
```

Comment:
instead of this dance should we just do this instead:

```js
const uniqueUrls = Array.from(new Set(resources));
```

Reply:
> const uniqueUrls = Array.from(new Set(resources));

unfortunately won't dedupe because it's an object, so they'll all be unique.

We could add it to the filter instead, the Boolean thing is pretty ugly :)

Reply:
we're moving this to replace `.filter(Boolean)`